Gemfile.lock

```diff
@@ -1149,4 +1149,4 @@ RUBY VERSION
    ruby 2.6.6p146
 
 BUNDLED WITH
-   1.17.3
+   1.17.3
```

Comment:
We shouldn't have Gemfile.lock and there needs to be a newline at end of file.

Reply:
Actually I think Gemfile.lock got added because there's no newline. If you add new line, it won't consider it changed.